### DISCUSSION TITLE
Last separator using Breadcrumb

### DISCUSSION THREAD

**Discussion by shyamajp** [twilio-labs/paste#1135] [Q&A] [answered]
Hello! 
In [documentation](https://paste.twilio.design/components/breadcrumb), it always has a separator after last child but in [storybook](https://twilio-labs.github.io/paste/?path=/story/components-breadcrumb--anchors), I do not see any last separator. Could you clarify which is the correct usage of it? 

The latter seems to be the "default" configuration. But if it is the former, I would like to know how to implement that with last separator. I tried adding last={false} hoping that it would add the last separator but it didn't work. Any tips?

Thank you in advance!

**Comment by richbachman** [comment, marked as answer]
Hi @shyamajp 

I think you may be referring to the breadcrumb we use at the top of our doc pages, i.e.: https://share.getcloudapp.com/RBu41rO5

If so, that is an oversight on our part, and causes confusion. We need to fix those to use our actual Breadcrumb component. They're using an older component we built when we first built the doc pages. We'll create a ticket to get that fixed.

The real Breadcrumb component does not have a separator after the last child, and is the correct usage. This example should help clarify things: https://paste.twilio.design/components/breadcrumb/#default-breadcrumb

Let us know if you have any other questions.

**Reply by shyamajp** [reply]
Thank you !!
